src/Microsoft.IIS.WebManager.Middleware/src/IISManagerExtension.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Rewrite;
+
+namespace Microsoft.IIS.WebManager.Middleware
+{
+    public static class IISManagerExtension
+    {
+        public static void UseIISWebManager(this IApplicationBuilder app, IHostingEnvironment env)
+        {
+            UseIISWebManager(app, env, string.Empty);   // string.Empty is not compile constant and cannot be used as optional parameter

```

Comment:
can you just use string requestPath = "" instead?

Reply:
sure